evap/evaluation/models.py

```diff
@@ -759,15 +771,20 @@ def can_publish_rating_results(self):
     def ready_for_editors(self):
         pass
 
-    @transition(field=state, source=State.PREPARED, target=State.EDITOR_APPROVED)
+    @transition(
+        field=state,
+        source=State.PREPARED,
+        target=State.EDITOR_APPROVED,
+        conditions=[lambda self: self.has_valid_main_language],
+    )
     def editor_approve(self):
         pass
 
     @transition(
         field=state,
         source=[State.NEW, State.PREPARED, State.EDITOR_APPROVED],
         target=State.APPROVED,
-        conditions=[lambda self: self.general_contribution_has_questionnaires],
+        conditions=[lambda self: self.general_contribution_has_questionnaires and self.has_valid_main_language],
```

Comment:
The issue says that staff users should be able to set "undecided". I think this change here would block that workflow?